oap-server/server-starter/src/main/resources/hierarchy-definition.yml

```diff
@@ -74,6 +74,9 @@ hierarchy:
     KAFKA: lower-short-name-with-fqdn
     PULSAR: lower-short-name-with-fqdn
 
+  CILIUM_SERVICE:
+    K8S_SERVICE: short-name
+
```

Comment:
Please update doc accordingly, https://skywalking.apache.org/docs/main/next/en/concepts-and-designs/service-hierarchy-configuration/

Reply:
updated.